LovrApp/Src/LovrApp_NativeActivity.cpp

```diff
@@ -172,14 +171,10 @@ void BridgeLovrUnpack(ovrRigidBodyPosef &HeadPose, BridgeLovrPose &Pose, BridgeL
 static void EglInitExtensions()
 {
 	const char * allExtensions = (const char *)glGetString( GL_EXTENSIONS );
-	__android_log_print(ANDROID_LOG_DEBUG, "LOVR", "In nativeactivity, version %s, extensions:\n%s\n", glGetString(GL_VERSION), allExtensions);
```

Comment:
Did you remove this for a reason?

Reply:
Ah, no good reason, let me put that back.